doc/ref_model/chapters/glossary.md

```diff
@@ -80,14 +78,24 @@ To help guide the reader, this glossary provides an introduction to the terminol
 
 - **Container Image**:	Stored instance of a container that holds a set of software needed to run an application.
 - **Container**:	A lightweight and portable executable image that contains software and all of its dependencies.
+>_*Note:*_ OCI defines **Container** as "An environment for executing processes with configurable isolation and resource limitations. For example, namespaces, resource limits, and mounts are all part of the container environment." 
+A **Container** provides operating-system-level virtualization by abstracting the “user space”. One big difference between **Containers** and **VM**s is that unlike VMs, where each **VM** is self-contained with all the operating systems components are within the **VM** package, containers "share" the host system’s kernel with other containers.
+
+- **Container Runtime**: reads the configuration files for a **Container** from a directory structure, uses that information to create a container, launches a process inside the container, and performs other lifecycle actions."
+>_*Note:*_ adapted from [OCI Glossary](https://github.com/opencontainers/runtime-spec/blob/master/glossary.md).
+
+- **Container Engine**: Software components used to create, destroy, and manage containers on top of an operating system.
+
 - **Pod**:	The smallest and simplest Kubernetes object. A Pod represents a set of running containers on your cluster. A Pod is typically set up to run a single primary container. It can also run optional sidecar containers that add supplementary features like logging.
-- **Kubernetes Cluster**: A set of machines, called nodes, that run containerised applications managed by Kubernetes. A cluster has at least one worker node and at least one master node.
-- **Kubernetes Master**:	The master node(s) manages the worker nodes and the pods in the cluster. Multiple master nodes are used to provide a cluster with failover and high availability.
+- **Kubernetes Cluster**: A set of worker machines, called nodes, that run containerised applications managed by Kubernetes. A cluster has at least one worker node and at least one master.
```

Comment:
Can we change back to original definition with the first use of "worker" - it is erroneous and not consistent with the Kubernetes definition: https://kubernetes.io/docs/reference/glossary/?fundamental=true#term-cluster

Reply:
I tried to resolve some comments but probably misunderstood.... 
Fine to delete the first worker. I hope this is what you suggested ;-)

Reply:
@tomkivlin Unfortunately, the glossary defines "node" as "A node is a worker machine in Kubernetes." Hence, using the term "master node" is confusing at best.

Reply:
Great, thanks @ulikleber

Reply:
@pgoyal01 but Kubernetes glossary also uses "master node": "Cluster: A set of machines, called nodes, that run containerized applications managed by Kubernetes. A cluster has at least one worker node and at least one master node. " (Link by @tomkivlin ).
If you have a better wording, please provide.

Reply:
Apologies @pgoyal01 I didn't see your comment when I resolved this conversation.  I think it is best to stick with the Kubernetes wording and if we think it needs updating to raise that as an Issue with the appropriate part of the glossary (all are here: https://github.com/kubernetes/website/tree/master/content/en/docs/reference/glossary).

One thing I have noticed however is that our definition here is missing the word `node` at the end of the definition. @ulikleber could you add that in please?

Reply:
@ulikleber we can use the standard definition as is but it is just inconsistent.  It is not important.
Any suggested definition would not conform to the Kubernetes Glossary. Possible non-standard alternatives include
"Cluster: A set of machines, called nodes and master, that run containerized applications managed by Kubernetes. A cluster has at least one worker node and at least one master. "

Reply:
What do others think? Should we follow Kubernetes glossary even while it is not 100% consistent, or should we fix the inconsistency as proposed? I am fine with both ways

Reply:
Since there was no further remark, I try whether people approve with @pgoyal01's proposal, which is very consistent, but doesn't have the word "node" at the end because in Kubernetes definition, the master is not a node.